backend/tests/lib/calculate/test_calculate_adequacy.py

```diff
@@ -21,62 +21,7 @@ def area_in_square_meters(polygon):
     return ops.transform(projection, polygon).area
 
 
-def test_fetch_service_area_polygons_db_no_radius():
```

Comment:
Can we still have an equivalent test somehow?

Reply:
Done.